src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+  const longtermDiscount = 50;
+  const shorttermDiscount = 20;
+
+  if (days >= 7) {
+    result = result - longtermDiscount;
+  } else if (days >= 3) {
```

Comment:
The number 3 here is a magic number. Please avoid using magic numbers. Instead, declare a constant at the top of your function.

e.g. `const shortTermDays = 3;`